src/smc-webapp/editor.coffee

```diff
@@ -1085,8 +1085,11 @@ class CodeMirrorEditor extends FileEditor
                 that.click_edit_button($(@).data('name'))
                 return false
 
+        # FIXME: removing the button here is pointless, since it will be added later again (somewhere)
         # FUTURE: implement printing for other file types
-        if @filename.slice(@filename.length-7) != '.sagews'
+        is_sagews = @filename.slice(@filename.length-7) != '.sagews'
```

Comment:
I know -- I lazily wrote that crappy code -- just encouraging you to fix it...

That said, the variable name has to be `is_not_sagews`, right?